packages/react/src/Overlay/Overlay.tsx

```diff
@@ -103,15 +103,6 @@ const StyledOverlay = toggleStyledComponent(
       max-width: calc(100vw - 2rem);
     }
 
-    &:where([data-variant='fullscreen']) {
```

Comment:
we have this in css now with ga, dont need to have it here as well

Reply:
I think let's keep this for now just in case, in theory the ga FF *could* still be turned off at any moment? The team can remove this when they remove the FF entirely from this component 🙏

Reply:
I thought we weren't adding adding css for new features? Based on https://github.com/primer/react/pull/5761#discussion_r2000030071

I recently added this css in https://github.com/primer/react/pull/5759, so thought I'll clean it up as well. yay/nay?

Reply:
yeah I think the issue here is the styles won't be added at all because if FF is not enabled the Overlay classname will not be added so these styles will never make it in (why you need that FF on that other test), the correct way of adding new features without sx styling is to always have it go to css modules, for example if you created a separate class and added it to both (enabled, not enabled) implementations